pkg/operator/staticpod/controllers.go

```diff
@@ -157,11 +165,13 @@ func (b *staticPodOperatorControllerBuilder) ToControllers() (*staticPodOperator
 			configMapClient,
 			secretClient,
 			eventRecorder,
-		)
+		))
+	} else {
+		errs = append(errs, fmt.Errorf("missing revisionController; cannot proceed"))
```

Comment:
why no hard return err?

Reply:
the original idea was to collect all errors I think, that is pre-existing.

Reply:
ic. The controller names are not necessary anymore.